pkg/config/config.go

```diff
@@ -165,6 +165,9 @@ type ContainersConfig struct {
 	// ShmSize holds the size of /dev/shm.
 	ShmSize string `toml:"shm_size,omitempty"`
 
+	//Tz sets the timezone inside the container
```

Comment:
```suggestion
	// TZ sets the timezone inside the container
```

I prefer to name it `TZ`

Reply:
@vrothberg Should I also name it such in my libpod PR?

Reply:
Yes, that would be great. Thanks!